parsl/providers/condor/condor.py

```diff
@@ -187,6 +197,15 @@ def submit(self, command, tasks_per_node, job_name="parsl.auto"):
 
         job_name = "parsl.{0}.{1}".format(job_name, time.time())
 
+        scheduler_options = self.scheduler_options
+        worker_init = self.worker_init
+        if self.mem_per_node is not None:
+            scheduler_options += 'RequestMemory = {}'.format(self.mem_per_node * 1024)
```

Comment:
Should there be `\n` on the end of this and line 206?

Reply:
Good catch-- I had changed that locally while testing and forgotten to push.

Reply:
It's fixed now